/*
#include <iostream> // Exceptions @ http://www.cplusplus.com/doc/tutorial/exceptions/
using namespace std;

int main () {
	try {
		try {
			throw 20;
		}
		catch (int param) {
			cout << "int exception" << endl;
			throw;
		}
	}
	catch (...) { cout << "default exception" << endl; }

	return 0;
}
*/
